scripts/install

```diff
@@ -122,7 +125,7 @@ ynh_mysql_connect_as "$db_name" "$db_pwd" "$db_name" \
 #=================================================
 
 rc_conf="${final_path}/config/config.inc.php"
-cp ../conf/config.inc.php "$rc_conf"
+cp -- ../conf/config.inc.php "$rc_conf"
```

Comment:
https://unix.stackexchange.com/questions/11376/what-does-double-dash-mean-also-known-as-bare-double-dash

Reply:
What's the (real) utility of this ?

Reply:
None in this particular case, but at least it enforces the use `--` as a norm. This has no drawback, so put a bit more security (without performance cost) than needed is not a technical problem.

Reply:
Your proposal makes sense but it seems not useful in many cases and I think it's a quite heavy practice in our case (apps packaging, used with few variables and not much risk of interfering options).

Reply:
Do you want that I use `--` only where a potential problem can occur?